website/docs/gitops-templates/annotations.mdx

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
For some reason I can't seem to find this page in the left nav of https://staging.docs.gitops.weave.works/revamp-template-docs/docs/next/gitops-templates/intro/

Reply:
huh 🤔 it's in the sidebars

 will take a look

Reply:
definitely there

![Screenshot 2023-03-03 at 15 06 24](https://user-images.githubusercontent.com/8898786/222754931-f88ade3c-9dc8-434b-8b07-52881b01ec22.png)